### PR TITLE
Create EFT, WIRE invoice from job

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4734

*Description of changes:*
- Changing job to create EFT and WIRE invoices in CFS
- Small null check for refund endpoint

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updated the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

**Review by nitheesh-aot — APPROVED** [review]

File: pay-api/src/pay_api/services/refund.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def create_refund(cls, invoice_id: int, request: Dict[str, str], **kwargs) -> No
 
         refund: RefundService = RefundService()
         refund.invoice_id = invoice_id
-        refund.reason = request.get('refund', None)
+        refund.reason = request.get('reason', None) if request else None
```
Comment (saravanpa-aot) [inline pay-api/src/pay_api/services/refund.py:140]: I use getAttr for this usage.. getAttr(request,reason,None) ..not sure which is better

**Review by saravanpa-aot — APPROVED** [review]
looks good

File: pay-api/src/pay_api/services/refund.py (hunk 1 above)
Comment (sumesh-aot) [reply pay-api/src/pay_api/services/refund.py:140]: looks like getattr doesn't work  for dict?  I have a dpath implementation for this, can use that one.

Comment (sumesh-aot) [reply pay-api/src/pay_api/services/refund.py]: Pushed change

**Pull request merged**